combinations/hash.tsv

```diff
@@ -250,3 +250,5 @@ muspinsim=1.1.0
 changeo=1.2.0,igblast=1.17.1,wget=1.20.1	quay.io/bioconda/base-glibc-busybox-bash:2.1.0	0
 cnvkit=0.9.9,samtools=1.15.1
 deeptools=3.5.1,samtools=1.15.1
+r-recetox-aplcms=0.9.4,r-base=4.1.0,r-arrow=4.0.1,r-dplyr=1.0.7	quay.io/bioconda/base-glibc-debian-bash:latest	0
+
```

Comment:
```suggestion
```
You might also want to open the PR from your fork to avoid the extra CI.

Reply:
thanks!